README.md

```diff
@@ -12,6 +12,7 @@ This library is friendly to PC and laptop execution, and optimized for resource
 
 Please follow the following blogs to setup your first hands-on experience with C++ and Python samples.
 
+* [How to Build OpenVINO™ GenAI](https://github.com/openvinotoolkit/openvino.genai/blob/master/src/docs/BUILD.md)
```

Comment:
I would suggest avoid mixing of our docs and Medium blogs. Let's have separate links to our Getting Started docs pages.

```suggestion
* [Introduction to OpenVINO™ GenAI](https://openvinotoolkit.github.io/openvino.genai/docs/getting-started/introduction)
* [Install OpenVINO™ GenAI](https://openvinotoolkit.github.io/openvino.genai/docs/getting-started/installation)
* [Build OpenVINO™ GenAI](https://github.com/openvinotoolkit/openvino.genai/blob/master/src/docs/BUILD.md)

Please follow the following blogs to setup your first hands-on experience with C++ and Python samples.

```